server/plugin/plugin.go

```diff
@@ -642,7 +642,7 @@ func (p *Plugin) storeGitHubToUserIDMapping(githubUsername, userID string) error
 
 func (p *Plugin) getGitHubToUserIDMapping(githubUsername string) string {
 	var data []byte
-	_ = p.client.KV.Get(githubUsername+githubUsernameKey, data)
+	_ = p.client.KV.Get(githubUsername+githubUsernameKey, &data)
```

Comment:
Good catch. Let's return the error and log it in `getGitHubToUsernameMapping` so this doesn't go unnoticed another time.

Reply:
I'm 1/5 that returning an error is better as it allows the caller to log with more context. Logging in `getGitHubToUserIDMapping` is also fine.

Reply:
> Logging in getGitHubToUserIDMapping is also fine

@hanzei We are going with this.